### DISCUSSION TITLE
Untrust player for all claims

### DISCUSSION THREAD

**Discussion by x1p** [GriefPrevention/GriefPrevention#834] [Ideas]
**Situation** Some players make a habit of getting trusted by multiple players and abuse trust.
**I want** to have a command to untrust that particular player on all existing claims.
**Because** When a player is caught grieving players, all unknowing players should be taken into protection by untrusting the griever on all players' claims. When the ban is expired and the formally griever can play again, no player should have to deal again with that griever because of a forgotten trust.

Optional questions:

**Is your feature request related to a problem? Please describe.**
Some players make a habit of getting trust on as much players as possible.

**Describe alternatives you've considered**
Downloading all GriefPreventionData/ClaimData, searching for the uuid of the griever and manually check all claims given the coordinates.

**Comment by jpcranford** [comment]
So basically, when a player is banned, automatically removed them from all trust lists?
